internal/view/helm.go

```diff
@@ -40,23 +42,33 @@ func (c *Helm) bindKeys(aa ui.KeyActions) {
 		ui.KeyShiftN: ui.NewKeyAction("Sort Name", c.GetTable().SortColCmd(nameCol, true), false),
 		ui.KeyShiftS: ui.NewKeyAction("Sort Status", c.GetTable().SortColCmd(statusCol, true), false),
 		ui.KeyShiftA: ui.NewKeyAction("Sort Age", c.GetTable().SortColCmd(ageCol, true), false),
-		ui.KeyV:      ui.NewKeyAction("Values Get", c.getValsCmd(false), true),
-		ui.KeyA:      ui.NewKeyAction("Values Get All", c.getValsCmd(true), true),
+		ui.KeyV:      ui.NewKeyAction("Values", c.getValsCmd(), true),
 	})
 }
 
-func (c *Helm) getHelmDao() *dao.Helm {
-	return model.Registry["helm"].DAO.(*dao.Helm)
-}
-
-func (c *Helm) getValsCmd(allValues bool) func(evt *tcell.EventKey) *tcell.EventKey {
+func (c *Helm) getValsCmd() func(evt *tcell.EventKey) *tcell.EventKey {
 	return func(evt *tcell.EventKey) *tcell.EventKey {
-		path := c.GetTable().GetSelectedItem()
-		vals, _ := c.getHelmDao().GetValues(path, allValues)
-		v := NewLiveView(c.App(), "Values", model.NewValues(c.GVR(), path, vals))
-		if err := v.app.inject(v); err != nil {
-			v.app.Flash().Err(err)
+		if path := c.GetTable().GetSelectedItem(); path != "" {
```

Comment:
Simplify! if there is nothing selected then bail out right away.

Reply:
Apologies, if I'm misunderstanding, but if the path == "" it should skip this block and bail and only require a single `return nil` yeah? I think I could do something like this:
```
		path := c.GetTable().GetSelectedItem()
		if path == "" {
			return nil
		}
		c.Values = model.NewValues(c.GVR(), path)
		v := NewLiveView(c.App(), "Values", c.Values)
		v.actions.Add(ui.KeyActions{
			ui.KeyV: ui.NewKeyAction("Toggle All Values", c.toggleValuesCmd, true),
		})
		if err := v.app.inject(v); err != nil {
			v.app.Flash().Err(err)
		}
		return nil
```
If you would prefer.

Reply:
yes!! better... in the case where we don't handle that event, might as well return the event vs nil.

Reply:
Gotcha, thanks!